.github/workflows/docker-release.yml

```diff
@@ -28,3 +28,25 @@ jobs:
           platforms: linux/amd64,linux/arm64
           push: true
           tags: awesometechnologies/synapse-admin:latest
+      - name: Calculate docker image tag
+        id: set-tag
+        run: |
+          case "${GITHUB_REF}" in
+              refs/heads/master|refs/heads/main)
+                  tag=latest
+                  ;;
+              refs/tags/*)
+                  tag=${GITHUB_REF#refs/tags/}
+                  ;;
+              *)
+                  tag=${GITHUB_SHA}
+                  ;;
+          esac
+          echo "::set-output name=tag::$tag"
+      - name: Build and push
```

Comment:
This should probably replace the "Build and push" step from line 24, right?

Reply:
No it will not replace https://github.com/sakkiii/synapse-admin/runs/3518227382
but yes it would be nice if we have different names. thanks

Reply:
But isn't it reduntant since it will also push to `latest`?

Reply:
Sorry it isn't very clear. In this pipeline there will be two push event for latest and tag calculated from set-tag step.

Reply:
The step "Calculate docker image tag" will return `latest` for pushes to the `master` branch. So your new step "Build and push" will also push to `latest`. This means we don't need the old "Build and push" step anymore.

Reply:
step "Calculate docker image tag" will not return `latest` as pipeline will only be triggered if this `[0-9]+\.[0-9]+\.[0-9]+'`  regex matches. So "Calculate docker image tag" will only return output like 0.X.X

Reply:
Ah, correct. You can add the branch `master` to the triggers as well.